src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx

```diff
@@ -0,0 +1,97 @@
+
+## At-mention your teammates and notify by email [#at-mention-your-teammates]
+
+Like other collaboration tools, when you at-mention (@) your teammates in a comment, they'll be notified by email. If you at-mention a teammate that isn't part of the preview or isn't using the new redesign, they won't have access to the discussion on New Relic. 
+
+## Share links and screenshots with Slack [#sync-with-slack]
+
+Whenever you start a new discussion, you can sync it with Slack so you can collaborate with all your teammates, even those that aren't on New Relic.
+
+Once the Slack app is configured and you are authenticated, select the Slack channel where you want your discussion synced. If you regularly use the same Slack channel it'll be prefilled so no extra clicks needed. From Slack, any teammate, even if they are not a New Relic user, can reply and their comment will be brought back into the thread on New Relic. Likewise subsequent replies added to the thread on New Relic will also be added to the thread on Slack.
+
+[Learn more about the Slack integration](https://docs-preview.newrelic.com/docs/collaboration-slack-integration).
+
+## Built-in screenshot sharing [#built-in-screenshots]
+
+With a new integrated screenshot selection tool you can attach screenshots to your comments. Click the “screenshot” icon, select the portion of the screen you want, and submit your comment. 
+
+## Close and re-open discussions [#close-reopen]
+
+Once a question is answered or issue is resolved you can mark a discussion as “Closed” via the more actions menu on the discussion thread. Similar to a Jira ticket your team can still add comments to the discussion. This will not automatically re-open a discussion but if you need to you can manually do so the same way as closing.
+
+<img
+  title="Close and reopen discussions "
+  alt="A screenshot of the UI that depicts closing discussions."
+  src={close_and_reopen_discussions}
+/>
+
+## Follow and unfollow discussions [#follow-unfollow]
+
+When you add a comment to a discussion or if you're at-mention you'll automatically be set to follow the discussion. This will be indicated by the bell icon on the discussion and on the discussions board. Following a discussion means you'll get email notifications and the Discussions board UMI (see next section) will increment when new comments are added to a discussion. If you want to unfollow a discussion that's also available via the more actions menu on the discussion (see screenshot in doc above).  Likewise, if there is a discussion you are interested in that you're not following but would like to, you can manually follow the same way. 
```

Comment:
Should say "if you're at-mentioned" and have a comma following the "at mention".

Reply:
The second two sentences are a bit unclear to me, is it saying "If you want to unfollow a discussion, you can do so via the more actions menu"? I think the second of the two sentences is only unclear because the first one is, but like before you can also cut out the Likewise.

Reply:
thanks!